hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -418,14 +418,14 @@
     {
       name: chip_sw_flash_init
       uvm_test_seq: chip_sw_flash_init_vseq
-      sw_images: ["//sw/device/tests/sim_dv:flash_init_test:0"]
+      sw_images: ["//sw/device/tests/sim_dv:flash_init_test:3"]
```

Comment:
```suggestion
      sw_images: ["//sw/device/tests/sim_dv:flash_init_test:0:test"]
```

Reply:
yeah @timothytrippel had suggested this way also, I actually though using a new enumeration was cleaner since it helped define the role better, but we could switch to this also.